.env

```diff
@@ -1,2 +1,3 @@
 NOTION_SECRET="secret_3nG9kNTECmX3gA7RFCoHCgYR8XAnQzAHv9rItdKXx1I"
-NOTION_DATABASE_ID="0aac587c9c8a4ad9b52c7a138efb3111"
\ No newline at end of file
+NOTION_DATABASE_ID="0aac587c9c8a4ad9b52c7a138efb3111"
+NEXT_PUBLIC_GOOGLE_API_KEY="AIzaSyAhYiRdgtBGRipCTmf8-qDY_ZaJjMZd86Y"
```

Comment:
Lol. 

We shall not commit `.env` (maybe is my fault cos I fix this in all the other projects) now is to time to fix and remember

Can create a .env.sample  (empty)
Can put the real .env variables in notion (credentials)
Remove this file (from git, not form your local) and ignore it `.gitignore`